File: _src/main/java/raccoonlang/parser/ast/FunctionBodyAstNode.java
package raccoonlang.parser.ast;

import raccoonlang.parser.Parser;
import raccoonlang.tokenizer.TokenType;

public class FunctionBodyAstNode {
    public static FunctionBodyAstNode parse(Parser parser) {
        FunctionBodyAstNode functionBodyAstNode = new FunctionBodyAstNode();

        parser.take(TokenType.OPEN_CURLY);
        parser.take(TokenType.CLOSE_CURLY);

        return functionBodyAstNode;
    }
}

File: Parser/AST/FunctionBodyAstNode.cs
namespace Raccoonlang.Parser.AST;

using Tokenizer;

public class FunctionBodyAstNode
{
    public static FunctionBodyAstNode Parse(Parser parser)
    {
        FunctionBodyAstNode node = new FunctionBodyAstNode();

        parser.Take(TokenType.OPEN_CURLY);
        // TODO: actually get body
        parser.Take(TokenType.CLOSE_CURLY);

        return node;
    }
}